lib/encoding/bufferreader.js

```diff
@@ -91,11 +91,16 @@ BufferReader.prototype.readUInt64BEBN = function() {
 };
 
 BufferReader.prototype.readUInt64LEBN = function() {
-  var buf = this.buf.slice(this.pos, this.pos + 8);
-  var reversebuf = BufferReader({
-    buf: buf
-  }).readReverse();
-  var bn = BN.fromBuffer(reversebuf);
+  var second = this.buf.readUInt32LE(this.pos);
+  var first = this.buf.readUInt32LE(this.pos + 4);
+  var combined = (first * 0x100000000) + second;
```

Comment:
Can't this lose precision? See:
```
> 0x100000000 * 0x10000000 
1152921504606847000
> 0x100000000 * 0x10000000 + 1
1152921504606847000
```
Both answers give the same value. Does this mean that if `second` is `0` or `1` and `first` is `0x10000000`, this would return the same value?